<?xml version="1.0" ?>

<launch>
    <!-- <include file="$(find qm_controllers)/launch/joy_teleop.launch"/> -->

    <node pkg="ocs2_legged_robot_ros" type="legged_robot_gait_command" name="legged_robot_gait_command"
              output="screen"/>

    <node pkg="qm_controllers" type="qm_target_trajectories_publisher" name="qm_target_trajectories_publisher"
          output="screen"/>

    <node name="gait_command_simulator" pkg="qm_controllers" type="gait_command_simulator" output="screen"/>

</launch>
